piper/ratbagd.py

```diff
@@ -28,7 +28,7 @@
 from enum import IntEnum
 from evdev import ecodes
 from gettext import gettext as _
-from gi.repository import Gio, GLib, GObject  # noqa
```

Comment:
Any reason for the change here? If you just want to import the latest `ratbagd.py` from libratbag, then import it completely (line 1 for example) and mention it in the commit description.

Reply:
Reason for this is that I copied over the file and that hunk went under my radar when viewing the commit diff. Thanks for catching it.